book/src/user/shielded-scan-grpc-server.md

```diff
@@ -0,0 +1,143 @@
+# Zebra Shielded Scanning gRPC Server
+
+## Get Started
+
+### Setup
+
+After setting up Zebra Shielded Scanning, add a `listen_addr` field to the shielded-scan configuration:
+
+```toml
+[shielded_scan]
+listen_addr = "127.0.0.1:8231"
+```
+
+Then, run `zebrad` to start the scan gRPC server.
+
+Making requests to the server will also require a gRPC client, the examples here use `grpcurl`, though any gRPC client should work.
+
+[See installation instructions for `grpcurl` here](https://github.com/fullstorydev/grpcurl?tab=readme-ov-file#installation).
+
+The types can be accessed through the `zebra-grpc` crate's root `scanner` module for clients in a Rust environment, and the [`scanner.proto` file here](https://github.com/ZcashFoundation/zebra/blob/main/zebra-grpc/proto/scanner.proto) can be used to build types in other environments.
+
+### Usage
+
+To check that the gRPC server is running, try calling `scanner.Scanner/GetInfo`, for example with `grpcurl`:
+
+```bash
+$ grpcurl -plaintext '127.0.0.1:8231' scanner.Scanner/GetInfo
+```
+
+The response should look like:
+
+```
+{
+  "minSaplingBirthdayHeight": 419200
+}
+```
+
+An example request to the `Scan` method with `grpcurl` would look like: 
+
+```bash
+grpcurl -plaintext -d '{ "keys": { "key": ["sapling_extended_full_viewing_key"] } }' '127.0.0.1:8231' scanner.Scanner/Scan
+```
+
+This will start scanning for transactions in Zebra's state and in new blocks as they're validated.
+
+Or, to use the scanner gRPC server without streaming, try calling `RegisterKeys` with your Sapling extended full viewing key, waiting for the scanner to cache some results, then calling `GetResults`.
+
+## gRPC Reflection
+
+To see all of the provided methods with `grpcurl`, try:
+
+```bash
+$ grpcurl -plaintext '127.0.0.1:8231' list scanner.Scanner
+```
+
+This will list the paths to each method in the `Scanner` service:
+```
+scanner.Scanner.ClearResults
+scanner.Scanner.DeleteKeys
+scanner.Scanner.GetInfo
+scanner.Scanner.GetResults
+scanner.Scanner.RegisterKeys
+```
+
+To see the the request and response types for a method, for example the `GetResults` method, try:
+
+
+```bash
+$ grpcurl -plaintext '127.0.0.1:8231' describe scanner.Scanner.GetResults \
```

Comment:
```suggestion
grpcurl -plaintext '127.0.0.1:8231' describe scanner.Scanner.GetResults \
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/8288/commits/6a1057ae5418a1afebdf89d895f5f3516cd6466f.